src/component/footer/Footer.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Link} from "react-router-dom";
+import "./footer.css";
+
+export default function Footer() {
+  const basics = [
+    { title: "About TMDB", link: " " },
+    { title: "Contact Us", link: " " },
+    { title: "Support Forums", link: " " },
+    { title: "System Status", link: " " },
+  ];
+
+  const getInvolved = [
+    { title: "Contribution Bible", link: " " },
+    { title: "Add New Movie", link: " " },
+    { title: "Add New TV Show", link: " " },
+  ];
+
+  const community = [
+    { title: "Guidelines", link: " " },
+    { title: "Discussions", link: " " },
+    { title: "Leaderboard", link: " " },
+  ];
+
+  const legal = [
+    { title: "Terms of Use", link: " " },
+    { title: "API Terms of Use", link: " " },
+    { title: "Privacy Policy", link: " " },
+    { title: "DMCA Policy", link: " " },
+  ];
+
```

Comment:
Try having all of these constants in a separate file called `constant.js` and import it in this file.